components/booking/SessionsList.tsx

```diff
@@ -14,7 +13,7 @@ interface sessionsProps {
 
 const SessionsList: React.FC<sessionsProps> = ({ type }) => {
   return (
-    <View style={styles.container}>
+    <>
```

Comment:
i think we dont need a fragment

Reply:
fixed on [708b523](https://github.com/ParkingSystemWithLPR/Napark-application/pull/28/commits/708b52375fdfe0ce0ba9dfad2ba18a861880de52)